scripts/parachain/local.sh

```diff
@@ -46,27 +46,60 @@ start_validator() {
     --$actor \
     --chain=./rococo-local-raw.json \
     --port=$port \
-    --rpc-port $rpc_port \
+    --rpc-port=$rpc_port \
     --tmp \
     --ws-port=$ws_port \
-    -lruntime=trace \
-    & node_pid=$!
+    -lruntime=trace
 }
 
-start_validator alice 30000 8000 9000
-start_validator bob 30001 8001 9001
+# Feel free to comment, add or remove validators. Just remember that #Validators > #Collators 
+
+start_validator alice 31000 8100 9100 & node_pid=$!
+start_validator bob 31001 8101 9101 & node_pid=$!
+start_validator charlie 31002 8102 9102 & node_pid=$!
+start_validator dave 31003 8103 9103 & node_pid=$!
+start_validator eve 31004 8104 9104 & node_pid=$!
+start_validator ferdie 31005 8105 9105 & node_pid=$!
 
 # Zeitgeist collators
 
-../release/zeitgeist \
-  --chain=./zeitgeist-raw.json \
-  --collator \
-  --parachain-id=$PARACHAIN_ID \
-  --port=31000 \
-  --rpc-port 8100 \
-  --tmp \
-  --ws-port=9100 \
-  -lruntime=trace \
-  -- \
-  --chain=./rococo-local-raw.json \
-  --execution=wasm
+start_collator() {
+  local author_id=$1
+
+  local collator_port=$2
+  local collator_rpc_port=$3
+  local collator_ws_port=$4
+
+  local relay_chain_port=$5
+  local relay_chain_rpc_port=$6
+  local relay_chain_ws_port=$7
+
+  ../release/zeitgeist \
+    --author-id=$author_id \
+    --chain=./zeitgeist-raw.json \
+    --collator \
+    --parachain-id=$PARACHAIN_ID \
+    --port=$collator_port \
+    --rpc-port $collator_rpc_port \
+    --tmp \
+    --ws-port=$collator_ws_port \
+    -lauthor-filter=trace \
+    -lauthor-inherent=trace \
+    -lruntime=trace \
+    -- \
+    --chain=./rococo-local-raw.json \
+    --execution=wasm \
+    --port=$relay_chain_port \
+    --rpc-port=$relay_chain_rpc_port \
+    --tmp \
+    --ws-port=$relay_chain_ws_port \
+    -lruntime=trace
+}
+
+# Feel free to comment, add or remove collators. Just remember that #Validators > #Collators
+
+start_collator 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY 30333 9933 9944 32000 8200 9200 & node_pid=$!
+start_collator 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty 30334 9934 9945 32001 8201 9201 & node_pid=$!
+start_collator 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY 30335 9935 9946 32002 8202 9202 & node_pid=$!
+start_collator 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty 30336 9936 9947 32003 8203 9203 & node_pid=$!
+start_collator 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty 30337 9937 9948 32004 8204 9204
```

Comment:
All the collators and validators spam the same console. I would prefer if only one validator and one collator write output to the console.

Reply:
I am open to suggestion. How to output one collator and one validator and keep everything else running in the background? It is also important to close all processes if the scripts exists

Reply:
NVM. Already figured it out